resource/js/scripts.js

```diff
@@ -250,6 +250,9 @@ function hideCrumbs() {
       $($crumbs[i]).addClass('hidden-path');
     }
     if ($('.restore-breadcrumbs').length === 0) {
+      // We attach the link to display all breadcrumbs after the zero-th element
+      // to force the browser to re-arrange the elements. If we moved that to the
+      // bottom, then we wouldn't have an easy way to re-arrange it at the top.
```

Comment:
@osma turns out `$crumbs[0]` was correct. But it's not the best way to have the link at the right top corner.

What we are doing, is injecting the element after the first item, to let the browser resize it. Previously it was doing it with `float: left` + `float: right`, and some width setting here and there.

But with a parent CSS flex element it gets trickier. For the future I think it would be good to restructure it so that the design was better enforced without relying on how the browser re-arranges elements. But not needed for now if the fix in this PR fixes it :+1:

Reply:
(Leaving the note to myself or other devs that work on this code)